### DISCUSSION TITLE
How can I use slate data in Flutter

### DISCUSSION THREAD

**Discussion by RevTpark** [ianstormtaylor/slate#5102] [Q&A]
I have a use case wherein we use Slate Editor on the web app(NextJS) and store the output directly in the database. The data can then be directly shown on the web app(i.e. no need to parse the stored data as slate does it for us). Now we also have a mobile app(Flutter) and we also want to display the stored data on the app. But the problem is that we have to parse that data and show it which is a complex task in itself. Is there a package or a better way how I can display the complex data structure in flutter? or how does the parser work in the backend so I can try to replicate the same logic?

**Comment by ahusnullin** [comment]
@RevTpark Have you found a solution to this problem?

**Reply by FullstackWEB-developer** [reply]
Please check this discuss
https://github.com/ueberdosis/tiptap/discussions/4332#discussioncomment-9391220

**Reply by Yusf1616** [reply]
#vaycasino
![IMG_20240929_035812](https://github.com/user-attachments/assets/1c78e09c-dba0-4531-9eae-5afd9bb527c4)

![IMG_20240929_035743](https://github.com/user-attachments/assets/dd32c8fe-5eb1-4cca-b6fc-781016238cd2)

**Comment by Yusf1616** [comment]
Hh
